.docs/content/1.concepts/5.rabbitmq.md

```diff
@@ -37,13 +37,13 @@ The credit system ensures that messages are not lost or dropped due to network c
 
 ## Limitations of RabbitMQ in ArmoniK
 
-ArmoniK is supposed to handle cases where a large number of tasks are submitted, waiting to be executed, but execution is slow. Therefore, task submission may outpace task execution. And we need to handle that properly. That's why we allow the publishers to outpace consumers. The current configuration of RabbitMQ flow control in ArmoniK.Core allow this to happen.  
+ArmoniK is designed to manage situations where a high volume of tasks are submitted, but if execution is slow, this might cause a backlog. In such cases, task submission can exceed the rate of execution, and we need to handle this imbalance effectively. To support this, we allow publishers to outpace consumers. The current RabbitMQ flow control configuration in ArmoniK.Coreis set up to enable this behavior.
```

Comment:
```suggestion
ArmoniK is designed to manage situations where a high volume of tasks are submitted, but if execution is slow, this might cause a backlog. In such cases, task submission can exceed the rate of execution, and we need to handle this imbalance effectively. To support this, we allow publishers to outpace consumers. The current RabbitMQ flow control configuration in ArmoniK. Core is set up to enable this behavior.
```